capstone/src/main/webapp/profile.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>My Profile</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+    <button class="explore-button" onclick="location.href='index.html';">Return to Explore</button>
+  </head>
+  <body>
+    <h1 class="heading">Megan Shi</h1>
+    <img src="images/profile.jpeg" class="profile-pic">
```

Comment:
I think images are supposed to have an alt, maybe just something like alt="profile picture" ?

Reply:
Ah, yeah I forgot about that. Thanks!